clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -103,6 +103,15 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`portability-template-virtual-member-function
+  <clang-tidy/checks/portability/template-virtual-member-function>` check.
+
+  Upon instantiating a template class, non-virtual member functions don't have 
```

Comment:
This should be single sentence. All additional explanations belong to documentation.

Reply:
How about the current one?